File: starter-bots/JavaBot/src/main/java/Enums/ObjectTypes.java
package Enums;

public enum ObjectTypes {
  PLAYER(1),
  FOOD(2),
  WORMHOLE(3),
  GAS_CLOUD(4),
  ASTEROID_FIELD(5);

  public final Integer value;

  ObjectTypes(Integer value) {
    this.value = value;
  }

  public static ObjectTypes valueOf(Integer value) {
    for (ObjectTypes objectType : ObjectTypes.values()) {
      if (objectType.value == value) return objectType;
    }

    throw new IllegalArgumentException("Value not found");
  }
}

File: starter-bots/KotlinBot/src/main/kotlin/galaxio/model/enums/ObjectTypes.kt
package galaxio.model.enums

enum class ObjectTypes(val value: Int) {
	PLAYER(1),
	FOOD(2),
	WORMHOLE(3),
	GAS_CLOUD(4),
	ASTEROID_FIELD(5),
	TORPEDO_SALVO(6),
	SUPERFOOD(7);

	companion object {
		fun valueOf(value: Int): ObjectTypes = values().first { it.value == value }
	}
}